### PR TITLE
[cw310] pins_cw310.xdc changes for BoB

### PR DESCRIPTION
**Pull request by hcallahan-lowrisc**
~~The only changed pins are IOB6-9 from USRDIP0-3 to PMOD2_IO5-8.~~
IOB7 : V7(USRDIP1) -> N23(PMOD2_IO5)
IOB8 : V8(USRDIP2) -> N26(PMOD2_IO6)
IOB9 : W9(USRDIP3) -> V7(USRDIP1)
IOB10 : V9(USRDIP4) -> V8(USRDIP2)
IOB11 : W8(USRDIP5) -> M20(PMOD2_IO7)
IOB12 : W10(USRDIP6) -> P25(PMOD2_IO8)
IOR5 : V11(USRDIP7) -> W9(USRDIP3)

### CODE REVIEW

**Comment by hcallahan-lowrisc** [comment]
@engdoreis When this is merged, you will need to update the mappings in i2c_testutils.c.

**Comment by a-will** [comment]
I retargeted the pins to align with hyperdebug (which, uh, also needed fixes, since I hooked up PMOD1 instead of PMOD2. Doh! That fix is in #18755 ). This will keep this FPGA variant aligned with the default test peripheral map.

I also adjusted the DIP switch mapping so the actual switch used didn't change for hello_usbdev.

File: hw/top_earlgrey/data/pins_cw310.xdc [hunk 1]
```diff
@@ -59,13 +59,13 @@ set_property -dict { PACKAGE_PIN A15  IOSTANDARD LVCMOS33 } [get_ports { IOB2  }
...
-set_property -dict { PACKAGE_PIN W10  IOSTANDARD LVCMOS18 } [get_ports { IOB12 }]; #  W10:USRDIP6                          DIP switches
+set_property -dict { PACKAGE_PIN U9   IOSTANDARD LVCMOS33 } [get_ports { IOB6  }]; #   U9:USRDIP0                          DIP switches
+set_property -dict { PACKAGE_PIN N23  IOSTANDARD LVCMOS33 } [get_ports { IOB7  }]; #  N23:PMOD2_IO5    EarlGrey:GPIO       (BoB):PMOD1_CSB7
+set_property -dict { PACKAGE_PIN N26  IOSTANDARD LVCMOS33 } [get_ports { IOB8  }]; #  N26:PMOD2_IO6    EarlGrey:GPIO       (BoB):PMOD1_RST
+set_property -dict { PACKAGE_PIN V7   IOSTANDARD LVCMOS33 } [get_ports { IOB9  }]; #   V7:USRDIP1                          DIP switches
+set_property -dict { PACKAGE_PIN V8   IOSTANDARD LVCMOS18 } [get_ports { IOB10 }]; #   V8:USRDIP2                          DIP switches
+set_property -dict { PACKAGE_PIN M20  IOSTANDARD LVCMOS18 } [get_ports { IOB11 }]; #  M20:PMOD2_IO7    EarlGrey:I2C0_SCL   (BoB):PMOD1_I2C_SCL
+set_property -dict { PACKAGE_PIN P25  IOSTANDARD LVCMOS18 } [get_ports { IOB12 }]; #  P25:PMOD2_IO8    EarlGrey:I2C0_SDA   (BoB):PMOD1_I2C_SDA
```
Comment (hcallahan-lowrisc) [inline hw/top_earlgrey/data/pins_cw310.xdc:68]: Unfortunately this arrangement puts SCL/SDA on a 1V8 bank. Is there another configuration that might work without too much churn?

Comment (a-will) [reply hw/top_earlgrey/data/pins_cw310.xdc:68]: Oh, that was just an accident--I didn't update the IOSTANDARD. This bank is 3.3 V.

Comment (a-will) [reply hw/top_earlgrey/data/pins_cw310.xdc]: Fixed.

Comment (hcallahan-lowrisc) [reply hw/top_earlgrey/data/pins_cw310.xdc]: oh great, ezpz :)

**Review by a-will — APPROVED** [review]

**Review by engdoreis — APPROVED** [review]

**Pull request merged**